rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -0,0 +1,228 @@
+use colored::*;
+use std::io::{self, Write};
+
+struct Step {
+    title: &'static str,
+    description: &'static str,
+}
+
+fn main() {
+    let steps = vec![
+        Step {
+            title: "Pick Release Candidate Commit",
+            description: "1. Go to https://github.com/dfinity/ic/actions/workflows/ci-main.yml?query=branch%3Amaster+event%3Apush+is%3Asuccess
+2. Find a recent commit with passing CI Main in the master branch
+3. Record this commit (e.g., post to Slack)
+
+Pre-built artifacts check:
+- Install aws tool if needed
+- List available files: 
+  aws s3 ls --no-sign-request s3://dfinity-download-public/ic/${COMMIT}/canisters/
+- Note: Our tools download from the analogous https://download.dfinity.systems/... URL",
+        },
+        Step {
+            title: "Determine Upgrade Targets",
+            description: "Determine which NNS canisters and/or SNS WASMs need to be upgraded/published.
+Only those with 'interesting' changes need to be released.
+
+Required checks:
+1. Run: ./testnet/tools/nns-tools/list-new-commits.sh
+2. Check Monday team sync meeting minutes at:
+   https://docs.google.com/document/d/1CPM1RlMz6UMSUQzqvdP7EDiLMomK4YeuEV7UnxQ9DAE/edit
+
+For SNS ledger suite (ledger, archive, and index canisters):
+- Consult Financial Integrations team
+- FI team should contact NNS team Friday morning about significant changes
+- FI team should provide the 'Features' section of proposals
+- This agreement is new - you may need to remind them
+- This applies to ledger, archive, and index canisters",
+        },
+        Step {
+            title: "Run NNS Upgrade Tests",
+            description: "1. Run NNS upgrade tests:
+   - Follow instructions in: testnet/tools/nns-tools/README.md#upgrade-testing-via-bazel
```

Comment:
This is only needed if you're not using a nightly build with a green check.

Reply:
Noted!